webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/EndpointsDefinition.scala

```diff
@@ -238,6 +238,10 @@ object NodeApi extends ApiModuleProvider[NodeApi] {
     val description = "List all accepted nodes with configurable details level"
     val (action, path)  = GET / "nodes"
   }
+  final case object GetNodesStatus extends NodeApi with ZeroParam with StartsAtVersion12 with SortIndex { val z = implicitly[Line].value
```

Comment:
Should we increase the version here ?? 12 is version 6.1

Reply:
done